app/models.py

```diff
@@ -38,6 +38,10 @@ class Lot(db.Model):
     name = db.Column(db.String, nullable=False)
     one_service_limit = db.Column(db.Boolean, nullable=False, default=False)
 
+    @property
+    def requires_brief(self):
```

Comment:
Yep agreed -`allow` is the word we never got to in the Slack conversation about this.